config/root-config.bat.in

```diff
@@ -75,7 +75,10 @@ for %%w in (%*) do (
       for %%f in (%features%) do (
          if %%f==!feature! (
             set res=%%f
-            set out=!out! yes
+            if not "!out!"=="" (
+               set out=!out! 
+            )
+            set out=!out!yes
```

Comment:
Doesn't this overwrite what was done in the `if` block? Or is this appending?

Reply:
It adds a space after `!out!`